xml/ha_rear.xml

```diff
@@ -517,17 +474,45 @@
      need <literal>no_root_squash</literal> as the
      <command>rear mkbackup</command> command runs as &rootuser;.
     </para>
-   </step>
-   <step>
+   </listitem>
+   <listitem>
     <para>
      Adjust the various <varname>BACKUP</varname> parameters in the
      configuration file <filename>/etc/rear/local.conf</filename> to make
      &rear; store the file backup on the respective NFS server. Find
      examples in your installed system under
      <filename>/usr/share/rear/conf/examples/SLE*-example.conf</filename>.
     </para>
-   </step>
-  </procedure>
+   </listitem>
+  </orderedlist>
+  </example>
+
+  <example xml:id="ex-ha-rear-config-sles-btrfs">
+    <title>Backing up Btrfs subvolumes with <command>tar</command></title>
+    <para>
+     Btrfs is set up with subvolumes, but <command>tar</command>
+     creates backups with the <option>--one-file-system</option> option, which excludes subvolumes.
+     Therefore, the subvolumes must be explicitly included in the &rear; configuration.
+    </para>
+    <para>
+     This <filename>/etc/rear/local.conf</filename> example is simplified from the example file
+     <filename>SLE12-SP2-btrfs-example.conf</filename>. It includes the whole
+     <filename>/var</filename> directory instead of specific <filename>/var</filename>
+     subdirectories. See the example file for details about the settings.
+    </para>
+<screen>OUTPUT=ISO
+BACKUP=NETFS
+BACKUP_OPTIONS="nfsvers=4,nolock"
+BACKUP_URL=nfs://<replaceable>host.&exampledomain;/path/to/rear/backup</replaceable>
+NETFS_KEEP_OLD_BACKUP_COPY=yes
+REQUIRED_PROGS=( "${REQUIRED_PROGS[@]}" snapper chattr lsattr )
+COPY_AS_IS=( "${COPY_AS_IS[@]}" /usr/lib/snapper/installation-helper /etc/snapper/config-templates/default )
+BACKUP_PROG_INCLUDE=( /root /boot/grub2/i386-pc /tmp /opt /var /boot/grub2/x86_64-efi /srv /usr/local )
```

Comment:
I'm thinking loudly here - since it is BASH, couldn't we just provide a comment to include all btrfs subvolumes ? Something like:

``` shell
$ findmnt -n -t btrfs -m -o TARGET,SOURCE | grep -Po '^(/[\w/-]+)(?=\s+'$(findmnt -nmo SOURCE / | grep -Po '^/dev/[\w/-]+')'.*)' | xargs
```

Reply:
Ah, the SLES 12 SP2 btrfs example file reads:

# You may use a command like
#   findmnt -n -r -o TARGET -t btrfs | grep -v '^/$' | egrep -v 'snapshots|crash'
# to generate the values:

So, could we add this into the docs? (Ignore then my previous comment.)